### ISSUE TITLE
Migrate the project for React

### ISSUE THREAD

**Issue by ramonbs** [akshitagupta15june/PetMe#305]
**Is your feature request related to a problem? Please describe.**
I came across an issue that was created 3-4 months ago, where someone had attempted to migrate to NextJS. Upon reviewing the entire project, I did not find any evidence of this migration. Therefore, I am reopening the issue to make it a reality.

**Describe the solution you'd like**
I would like to fully migrate the project to React. This will make the application more fluid and user-friendly for new developers who want to contribute.

**Describe alternatives you've considered**
While I understand the benefits of using NextJS, I am not a big fan of it as it seems to bring more problems than solutions in my experience (especially with the upcoming node server). Another alternative could be Remix, but I do not have enough knowledge about it to comment on it.

**Additional context**
I would like to implement some rules, particularly clean code. For this, I request the implementation of ESLint and StyleLint, which will make the code much more readable. As for the code itself, I propose using functional components and a simple, organized data structure. I have already finished the main component, the homepage. If this issue receives a green signal, I will submit the results as soon as possible.

**Comment by akshitagupta15june** [comment]
You can go for it @ramonbs

**Comment by ramonbs** [comment]
Okay, @raslene5! I'll open a pull request right now and we can get started! =) There's a lot to do!!

@akshitagupta15june, thank you! Could you create a new branch on the original repository, maybe called "react/dev"? I'll open the first pull request to it.

**Comment by damiend97** [comment]
@ramonbs I'd also love to lend a hand in this migration. I'll check back up on this and see if the ball gets rolling.

**Comment by akshitagupta15june** [comment]
Ok i have created a new branch react/dev

**Comment by ramonbs** [comment]
Everyone is more than welcome to help with the project migration! I have already made the first pull request @damiend97  @raslene5. @akshitagupta15june, where do you suggest I could leave a TODO list for our migration? For now, I have some ideas on what we should focus on.

**Comment by ramonbs** [comment]
Yes, I have Discord, that's my nickname and tag: Luster (Ramon)#9866

**Comment by akshitagupta15june** [comment]
Cool I have sent u a request from ScorpionDoyenne#1756

**Comment by damiend97** [comment]
Damien/Nora#0001

**Comment by akshitagupta15june** [comment]
@damiend97 your I'd is invalid

**Comment by akshitagupta15june** [comment]
So I am assigning this issue to all

**Comment by damiend97** [comment]
> @damiend97 your I'd is invalid

Fixed. Sorry

**Comment by karthiknadar1204** [comment]
Is this issue still available?i would like to work on it,please assign it to me

**Comment by karthiknadar1204** [comment]
will begin the work asap @ramonbs

**Comment by leanjunio** [comment]
Is there still room to contribute to this?
